### PR TITLE
Component Swapping in extension and Github content bug

### PR DESCRIPTION
**Pull request by shweta-fable**
- [x] Add component swapping support to extension
- [x] Fix bug long github content not shared

### CODE REVIEW

File: packages/mdx-preview-iframe/src/content.ts [hunk 1]
```diff
@@ -112,4 +105,105 @@ export default class ErrorBoundary extends Component{
...
     }
   }
-`;
\ No newline at end of file
+`;
+
+export const appContext = `
+import React, { createContext, useContext, useState, useEffect } from 'react';
+import { useSearchParams } from "react-router-dom";
+import manifest from './manifest.json'
```
Comment (siddhigate12) [inline packages/mdx-preview-iframe/src/content.ts:113]: instead of hard coding it here, we can use the build-static-files script to generate it

Comment (shweta-fable) [reply packages/mdx-preview-iframe/src/content.ts]: done

File: packages/common/src/utils.ts [hunk 2]
```diff
@@ -420,3 +424,27 @@ const traverseConfig = (config: Config, path: string[]): any => {
   path.forEach(key => obj = obj[key]);
   return obj;
 };
+
+export const getLayoutContents = (staticLayoutPath: string, distLoc: string): LayoutData[] => {
```
Comment (siddhigate12) [inline packages/common/src/utils.ts:428]: if staticLayoutPath isn't required here, u can remove it.

Comment (shweta-fable) [reply packages/common/src/utils.ts]: done

**Review by siddhigate12 — APPROVED** [review]

**Pull request merged**